src/app/retrospective/retrospect-task-modal/retrospect-task-modal.component.ts

```diff
@@ -169,8 +180,25 @@ export class RetrospectTaskModalComponent implements OnDestroy {
                     }
                 }
             );
+        return getTaskMemberSummary$;
     }
 
+    refreshIssueMemberSummaryAgGrid() {
+        this.gridApi.showLoadingOverlay();
+        // To clear the current content of the grid
+        this.gridApi.setRowData(null);
+        const getTaskMemberSummary$ = this.getSprintTaskMemberSummary(true);
+        getTaskMemberSummary$.subscribe(
+            () => {
+                this.gridApi.hideOverlay();
+            },
+            () => {
+                this.gridApi.hideOverlay();
```

Comment:
Use complete callback if we have to perform an action in both next and error callbacks.

Reply:
`() => this.gridApi.hideOverlay()`